InformedSolutions/JAQU-CAZ-MOD-UI
# [VCCS-532] Handle cognito authorized_list_type

## Motivation and Context


[JIRA](https://eaflood.atlassian.net/browse/VCCS-532)
## Description

How to test?
1. Add custom attribute `authorized-list-type` on AWS Cognito
2. Add read and write access in **App cliens**
3. Go to Users - Import - Download csv headers 
4. Add [required values](https://eaflood.atlassian.net/browse/NTR-376) and value for custom attribute(e.g. `green` or `white`)
5. Import users as was described [here](https://eaflood.atlassian.net/browse/NTR-376)
6. Now log in with a new create user
7. Upload csv file to s3
8. Check file content type on s3 (should be added `MOD_GREEN_LIST`)

## How Has This Been Tested?




## Screenshots (if appropriate):
![result](https://user-images.githubusercontent.com/24584219/64036404-6a709d00-cb53-11e9-938b-9ebb1e0d08b2.png)
![root_page](https://user-images.githubusercontent.com/24584219/64101766-db3fd100-cd6e-11e9-97df-0fe7cee5eb81.png)
## Checklist:


- [x] Includes a link to an issue (if apply)
